ocaml/libs/tracing/tracing.ml

```diff
@@ -602,9 +605,7 @@ let get_tracer ~name =
         Hashtbl.fold (fun _k v acc -> v :: acc) tracer_providers []
```

Comment:
This is probably not directly relevant, but this line looks the same as the `get_tracer_providers` function, can we use that instead?

Reply:
Yes, thank you!